docs/en_US/reference/emqxBroker.md

```diff
@@ -0,0 +1,445 @@
+
+  If specified, the pod's tolerations.
+
+  *The pod this Toleration is attached to tolerates any taint that matches the triple <key,value,effect> using the matching operator .*
+
+  - **tolerations.key** (string)
+
+    Key is the taint key that the toleration applies to. Empty means match all taint keys. If the key is empty, operator must be Exists; this combination means to match all values and all keys.
+
+  - **tolerations.operator** (string)
+
+    Operator represents a key's relationship to the value. Valid operators are Exists and Equal. Defaults to Equal. Exists is equivalent to wildcard for value, so that a pod can tolerate all taints of a particular category.
+
+    Possible enum values:
+
+    - `"Equal"`
+    - `"Exists"`
+
+  - **tolerations.value** (string)
+
+    Value is the taint value the toleration matches to. If the operator is Exists, the value should be empty, otherwise just a regular string.
+
+  - **tolerations.effect** (string)
+
+    Effect indicates the taint effect to match. Empty means match all taint effects. When specified, allowed values are NoSchedule, PreferNoSchedule and NoExecute.
+
+    Possible enum values:
+
+    - `"NoExecute"` Evict any already-running pods that do not tolerate the taint. Currently enforced by NodeController.
+    - `"NoSchedule"` Do not allow new pods to schedule onto the node unless they tolerate the taint, but allow all pods submitted to Kubelet without going through the scheduler to start, and allow all already-running pods to continue running. Enforced by the scheduler.
+    - `"PreferNoSchedule"` Like TaintEffectNoSchedule, but the scheduler tries not to schedule new pods onto the node, rather than prohibiting new pods from scheduling onto the node entirely. Enforced by the scheduler.
+
+  - **tolerations.tolerationSeconds** (int64)
+
+    TolerationSeconds represents the period of time the toleration (which must be of effect NoExecute, otherwise this field is ignored) tolerates the taint. By default, it is not set, which means tolerate the taint forever (do not evict). Zero and negative values will be treated as 0 (evict immediately) by the system.
+
+  - **nodeSelector** (map[string]string)
+
+    NodeSelector is a selector which must be true for the pod to fit on a node. Selector which must match a node's labels for the pod to be scheduled on that node. More info: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
+
+  - **nodeName** (string)
+
+    NodeName is a request to schedule this pod onto a specific node. If it is non-empty, the scheduler simply schedules this pod onto that node, assuming that it fits resource requirements.
+
++ **env** ([]EnvVar)
+
+  *Patch strategy: merge on key `name`*
+
+  List of environment variables to set in the container. Cannot be updated.
+
+  *EnvVar represents an environment variable present in a Container.*
+
+  - **env.name** (string), required
+
+    Name of the environment variable. Must be a C_IDENTIFIER.
```

Comment:
We'd better give the rules for how to convert the key such as "log.level" in emqx docs to env which can be parsed correctly in container.